hack/vendor.sh

```diff
@@ -25,13 +25,16 @@ clone git github.com/armon/go-metrics eb0af217e5e9747e41dd5303755356b62d28e3ec
 clone git github.com/hashicorp/go-msgpack 71c2886f5a673a35f909803f38ece5810165097b
 clone git github.com/hashicorp/memberlist 9a1e242e454d2443df330bdd51a436d5a9058fc4
 clone git github.com/hashicorp/serf 7151adcef72687bf95f451a2e0ba15cb19412bf2
-clone git github.com/docker/libkv ea7ff6ae76485ab93ac36799d3e13b1905787ffe
+clone git github.com/docker/libkv 22a9d7c7674819859798e3e823245f131520081e
 clone git github.com/vishvananda/netns 604eaf189ee867d8c147fafc28def2394e878d25
 clone git github.com/vishvananda/netlink 4b5dce31de6d42af5bb9811c6d265472199e0fec
 clone git github.com/BurntSushi/toml f706d00e3de6abe700c994cdd545a1a4915af060
 clone git github.com/samuel/go-zookeeper d0e0d8e11f318e000a8cc434616d69e329edc374
 clone git github.com/deckarep/golang-set ef32fa3046d9f249d399f98ebaf9be944430fd1d
-clone git github.com/coreos/go-etcd v2.0.0
+clone git github.com/coreos/etcd ba949ae2be9a2dee53e09c5c0ad365233a4a4711
```

Comment:
They do: it's the [`client/` subdirectory of the `coreos/etcd` repo](https://github.com/coreos/etcd/tree/master/client), so I believe this line is fine. However, we drag another big one with it (such as `github.com/ugorji/go` below).

Also, the fact that imports are rewritten in etcd required @dhiltgen to have this `fiximports` function.